cs/index.json

```diff
@@ -40,7 +40,8 @@
     "progress" : "Postup ",
     "stop_benchmark" : "Zastavit Benchmark",
     "internet_status_connected" : "Připojeno",
-    "internet_status_disconnected" : "Odpojeno"
+    "internet_status_disconnected" : "Odpojeno",
+    "benchmark_algorithm_fail": "Benchmark fail for algorithms: {{algorithms}}"
```

Comment:
"benchmark_algorithm_fail": "Benchmark následujících algoritmů selhal: {{algorithms}}"

Reply:
@czjohns updated and thanks.